addons/main.go

```diff
@@ -146,6 +147,17 @@ func main() {
 		os.Exit(1)
 	}
 
+	antreaConfigReconciler := &antrea.AntreaConfigReconciler{
```

Comment:
Remove from main, otherwise controller will crash because CRD is not installed. See https://github.com/vmware-tanzu/tkg-devel/issues/234

Reply:
Fixed